tests/test_endpoints.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+import json
+from app import create_app
+import sys
+sys.path.append("..")
+
+
+@pytest.yield_fixture
+def app():
+    app = create_app("testing")
+    yield app
+
+
+@pytest.fixture
+def test_cli(loop, app, sanic_client):
+    return loop.run_until_complete(sanic_client(app))
+
+
+async def test_index(test_cli):
+    resp = await test_cli.get('/')
+    assert resp.status == 200
+
+
+async def test_getLanguages_with_no_query_param(test_cli):
+    resp = await test_cli.get('/languages')
+    resp_json = await resp.json()
+    assert resp_json["total_results"] == 0
+    assert resp_json["page"] == 1
+    assert resp.status == 200
+
+
+async def test_getLanguages_with_more_query_param(test_cli):
```

Comment:
In general, you want to make the test name tell the intention of what you are testing. When I read the test name and the content of the test I am not sure I understand what is being tested. your error message says expecting less than 3 parameters but the query look completely legit to me. I think it is ok to fetch the second page of all local languages with every page being 50 elements. Please let me know if I am missing something.

Reply:
Do you have a suggestion for a good function name ? 
`limit` is not a valid argument for the request. I will renamed the argument to avoid confusion